src/library_memfs.js

```diff
@@ -274,6 +274,9 @@ mergeInto(LibraryManager.library, {
       },
 
       // Writes the byte range (buffer[offset], buffer[offset+length]) to offset 'position' into the file pointed by 'stream'
+      // canOwn: A boolean that tells if this function can take ownership of the passed in buffer from the subbuffer portion
+      //         that the typed array view 'buffer' points to. The underlying ArrayBuffer can be larger than that, but
+      //         canOwn=true will not take ownership of the portion outside the bytes addressed by the view.
```

Comment:
Ok, explained what taking ownership means, but used a slightly different wording to try to be precise.